org.testeditor.web.backend.xtext/src/main/java/org/testeditor/web/backend/xtext/index/IndexServiceClient.xtend

```diff
@@ -80,6 +93,24 @@ class IndexServiceClient implements IGlobalScopeProvider {
 				Entity.text(null)
 		}
 	}
+	
+	private def buildRequest(WebTarget target)
+	{
+		val indexServiceRequest = target.request(MediaType.APPLICATION_JSON)
+		val contextRequest = requestProvider.get 
+		if (contextRequest !== null) {
+			val authorizationHeader = contextRequest?.getHeader(AUTHORIZATION)
+			if (authorizationHeader !== null && authorizationHeader != "") {
+				return indexServiceRequest.header(AUTHORIZATION, authorizationHeader)
+			}
+			else {
+				logger.info("Context request carries no authorization header. Request to index service will be sent without authorization header.")
```

Comment:
should be a warning, too (since authorization is the default)

Reply:
OK.

… more importantly, though, the correct level is not currently being checked for in the corresponding unit tests.